.github/label-commenter-config.yml

```diff
@@ -0,0 +1,15 @@
+labels:
+  - name: needs-docs
+    labeled:
+      pr:
+        body: |
+          Thanks for your pull request, before this can be merged - corresponding documentation for your module is required:
+          - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
+          - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
+          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
+        action: open
+    unlabeled:
+      issue:
```

Comment:
Does this need to be `pr` to match the block above?
```suggestion
      pr:
```

Reply:
In this case, the setting is also used for `pr`. If the `thank you` message is for pull requests, the suggested change looks better.

cf. https://github.com/peaceiris/actions-label-commenter/blob/547c566e1705f7f1df682811f10672345406c49f/src/main.ts#L98

Reply:
Sorry, I released v1.3.2 now. if the setting is undefined, the action will do nothing.